charts/gateway-helm/values.tmpl.yaml

```diff
@@ -36,6 +36,8 @@ config:
       controllerName: gateway.envoyproxy.io/gatewayclass-controller
     provider:
       type: Kubernetes
+      kubernetes:
```

Comment:
this will no longer be needed, since `leaderElection` will be enabled by default